google/fingerprinters/web/scripts/updater/community/solr/app/docker-compose.yaml

```
@@ -0,0 +1,8 @@
+services:
+  solr:
+    image: solr:${SOLR_VERSION}
+    ports:
+     - "8983:8983"
+    command:
+      - solr-precreate
+      - gettingstarted
```

```suggestion
      - gettingstarted
    healthcheck:
      test: curl --fail http://localhost:8983/ || exit 1
      interval: 10s
      timeout: 10s
      retries: 3
      start_period: 30s
```
You need to specify the healthcheck like this so that the `--wait` argument has a way to tell if the service is healthy.